src/main/java/ratpack/zipkin/ServerTracingModule.java

```diff
@@ -115,7 +117,7 @@ public LocalTracer localTracer(final Brave brave) {
   
   public static class Config {
     private String serviceName = "unknown";
-    private SpanCollector spanCollector;
+    private Reporter<Span> spanReporter;
```

Comment:
Is upstream missing these changes?  It'd be nice to make a pull request there as well.